src/python/pants/backend/jvm/tasks/coursier/coursier_subsystem.py

```diff
@@ -36,14 +36,14 @@ def register_options(cls, register):
     register('--fetch-options', type=list, fingerprint=True,
              help='Additional options to pass to coursier fetch. See `coursier fetch --help`')
     register('--bootstrap-jar-url', fingerprint=True,
-             default='https://dl.dropboxusercontent.com/s/t8zz2nupqkv2sfv/coursier-cli-1.0.2.e943c8069180d22f352878949f9522ad91d89026.jar?dl=0',
+             default='https://dl.dropboxusercontent.com/s/zwh074l9kxhqlwp/coursier-cli-1.1.0.cf365ea27a710d5f09db1f0a6feee129aa1fc417.jar?dl=0',
```

Comment:
was surprised to see this default option pointing to an arbitrary dropbox URL.

is there no better location to host this jar?

Reply:
Temp solution. Will be part of the cleanup to move to binutil once we get this out of the door.

Reply:
@wisechengyi : Sorry, I've asked this before, but: is there any plan for upstream to begin publishing the CLI? Can we link to it as a TODO/github issue?

Reply:
It is being published http://www.maven.org/#search%7Cgav%7C1%7Cg%3A%22io.get-coursier%22%20AND%20a%3A%22coursier-cli_2.12%22. We could check the most recent version to verify it includes our changes and switch to that.

Reply:
Yep. plan to take a look at that soon.